src/cdomains/apron/relationDomain.apron.ml

```diff
@@ -191,26 +189,11 @@ struct
     let tr = QCheck.pair (RD.arbitrary ()) (PrivD.arbitrary ()) in
     QCheck.map ~rev:to_tuple of_tuple tr
 
-  let bot () = {rel = RD.bot (); priv = PrivD.bot ()}
-  let is_bot {rel; priv} = RD.is_bot rel && PrivD.is_bot priv
-  let top () = {rel = RD.top (); priv = PrivD.bot ()}
```

Comment:
We have some top/bot confusion here and would need to do this manually here.

Reply:
#1463 fixed a similar thing in base. This is probably the same typo copied.